API_register/models.py

```diff
@@ -21,6 +21,13 @@ def get_classes(self):
         return self.classes_set.all()
 
 
+class AnnouncementManager(models.Manager):
```

Comment:
This manager might be reused for other models.

Why don't you go further and use an abstract model for that?

```python
class HiddenModel(models.Model):
    hidden = models.BooleanField(default=False)
    objects = HiddenManager()

    class Meta:
        abstract = True

```